File: 2057_smallest_index_with_equal_value/Solution.cs
// LeetCode 2057 - Smallest Index With Equal Value
// https://leetcode.com/problems/smallest-index-with-equal-value/

public class Solution {
    public int SmallestEqual(int[] nums) {
        for (int i = 0; i < nums.Length; i++)
            if (i % 10 == nums[i]) return i;
        return -1;
    }
}

File: 2057_smallest_index_with_equal_value/solution.py
# LeetCode 2057 - Smallest Index With Equal Value
# https://leetcode.com/problems/smallest-index-with-equal-value/

from typing import List


class Solution:
    def smallestEqual(self, nums: List[int]) -> int:
        for i, v in enumerate(nums):
            if i % 10 == v:
                return i
        return -1
